include/deal.II/base/parameter_handler.h

```diff
@@ -851,52 +851,59 @@ class ParameterHandler : public Subscriptor
    */
   enum OutputStyle
   {
+    /**
+     * Write input for ParameterHandler without comments or changed default
+     * values.
+     */
+    Short = 0x0001,
+
+    /**
+     * TODO
+     */
+    Unsorted = 0x0002,
+
     /**
      * Write human readable output suitable to be read by ParameterHandler
      * again.
      */
-    Text = 1,
+    Text = 0x0010,
 
     /**
      * Write parameters as a LaTeX table.
      */
-    LaTeX = 2,
+    LaTeX = 0x0020,
+
     /**
      * Write out declared parameters with description and possible values.
```

Comment:
sure, but don't you see the difficulties in documentation? Since we don't have a struct design, which would more intuitively show to which part this parameter belongs to, we should document it. My point is that this parameter can be (mis-)understood as the boolean `Short` (or its opposite).